### PR TITLE
[DOCFIX] Add compatibility doc

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: docs/en/Compatibility.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Alluxio journal needs to be upgraded manually when upgrading Alluxio cluster to 
+{{site.ALLUXIO_RELEASED_VERSION}} from any Alluxio cluster deployed with vesion < 1.5.0 by running
+the following command in the home directory of the new cluster. 
+
+```bash
+$ ./bin/alluxio upgradeJournal -journalDirectoryV0 YourJournalDirectoryV0 
+```
+
+It is strongly recommended to backup the old journal (journal v0) before running the journal
+upgrade tool to avoid losing any data in case of failures.
```
Comment (calvinjia) [inline docs/en/Compatibility.md]: Could you add the command to do so (people may assume there is a specific command)?

Comment (peisun1115) [inline docs/en/Compatibility.md]: done

**Comment by peisun1115** [comment]
@calvinjia PTAL? thanks.

File: docs/en/Compatibility.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+title: Compatibility 
+nickname: Compatibility 
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+This document lists features introduced in {{site.ALLUXIO_RELEASED_VERSION}} that are incompatible
```
Comment (calvinjia) [inline docs/en/Compatibility.md]: This page isn't specific to the version anymore right?

Comment (peisun1115) [inline docs/en/Compatibility.md]: done

**Pull request merged**